src/styles/index.scss

```diff
@@ -1,3 +1,59 @@
+@import './utils/animations';
+
 body {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
   margin: 0;
 }
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin #000 dotted;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), color values should be stored in variables for easier maintenance and to improve readability. Consider using a variable for the color value '#000'.